Gemfile.lock

```diff
@@ -460,7 +465,7 @@ GEM
       addressable (~> 2.3)
     letter_opener (1.6.0)
       launchy (~> 2.2)
-    libv8 (3.16.14.19)
+    libv8 (6.7.288.46.0)
```

Comment:
According to the comment in https://github.com/openfoodfoundation/openfoodnetwork/pull/2867/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R96, This version is just one patch below the problematic one.